java/pom.xml

```diff
@@ -130,6 +130,11 @@
         <artifactId>jaxb-impl</artifactId>
         <version>${jaxb.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.5</version>
+      </dependency>
```

Comment:
What's gson for?

Reply:
It's actually needed for the next PR (serializing handle with json and pass it to Python), forgot to remove it.